x/twap/types/utils.go

```diff
@@ -75,3 +76,13 @@ func SpotPriceTimesDuration(sp sdk.Dec, timeDelta time.Duration) sdk.Dec {
 func AccumDiffDivDuration(accumDiff sdk.Dec, timeDelta time.Duration) sdk.Dec {
 	return accumDiff.QuoInt64(int64(timeDelta))
 }
+
+func LexicographicalOrderDenoms(denom0, denom1 string) (string, string, error) {
```

Comment:
Also, since we don't seem to be using this anywhere outside of twap, does it make sense to have it directly in the `x/twap` to avoid leaking logic to the exported scope?

Reply:
Agreed on creating a test for it. This is used in CreateTwapRecord as well though which is in utils.go. Should I keep it in utils because of this?

Reply:
I do think we should probably move CreateTwapRecord to the main keeper in logic.go, its basically only used there.

I am in favor of keeping lexicographic ordering of denoms here personally tho, its a helper that someone working with TWAPs externally may want

Reply:
Moved CreateTwapRecord to logic.go!